docs/getting-started.md

```diff
@@ -63,6 +63,8 @@ curl localhost:10101/index/repository -X POST
 ``` response
 {"success":true}
 ```
+Note: 
+The index name must be 64 characters or less in length and must  be a lower case string: a lowercase letter followed by alphanumeric characters and/or `_-`. 
```

Comment:
This is a bit wordy and uses both "lowercase" and "lower case" (lowercase seems to be correct, though I had to look it up)

Maybe:
"The index name must be 64 characters or less, start with a letter, and consist only of lowercase alphanumeric characters or `_-`."